src/main/java/org/broadinstitute/hellbender/tools/spark/ParallelCopyGCSDirectoryIntoHDFSSpark.java

```diff
@@ -26,17 +26,47 @@
 import java.util.stream.Collectors;
 
 /**
- * This tool uses Spark to do a parallel copy of either a file or a directory from GCS into HDFS.
- * Files are divided into chunks of size equal to the HDFS block size (with the exception of the final
- * chunk) and each Spark task is responsible for copying one chunk. To copy all of the files in a GCS directory,
- * provide the GCS directory path, including the trailing slash. Directory copies are non-recursive so
+ * Stage a Google Cloud Storage file or directory into an HDFS file system.
```

Comment:
ChrisN merged the new program groups this afternoon.
So replace `SparkProgramGroup` with `OtherProgramGroup` after rebasing.